ledger-core/virtual/integration/contract_calls_test.go

```diff
@@ -505,3 +508,148 @@ func TestVirtual_CallMethodFromConstructor(t *testing.T) {
 		})
 	}
 }
+
+func TestVirtual_CallContractFromContract_RetryLimit(t *testing.T) {
+	t.Log("CXXXX")
```

Comment:
strange id

Reply:
it is draft, i don't add it to testrail yet